utils/src/main/kotlin/DirectoryStash.kt

```diff
@@ -37,11 +37,13 @@ fun stashDirectories(vararg directories: File): Closeable = DirectoryStash(setOf
  * A Closable class which moves the given directories to temporary directories on initialization. On close, it deletes
  * any conflicting existing directories in the original location and then moves the original directories back.
```

Comment:
The documentation needs to be updated to make clear that not only "conflicting" directories get deleted.

Reply:
Done.